TemplePlus/python/python_integration_d20_action.cpp

```diff
@@ -133,9 +133,9 @@ PYBIND11_EMBEDDED_MODULE(tpactions, m) {
 		return TRUE;
 	});
 
-	m.def("get_cur_seq", []()->ActnSeq &{
-		return **actSeqSys.actSeqCur;
-	});
+	m.def("get_cur_seq", []()->ActnSeq *{
+		return *actSeqSys.actSeqCur;
+	}, py::return_value_policy::reference );
```

Comment:
I've changed get_cur_seq to return a reference rather than a copy. I vaguely recall someone needed this change, pinging @dolio @Sagenlicht @anatoliy-savchak @doug1234

Reply:
It wasn't me, though I do use get_cur_seq in quite a few python actions.

Reply:
> Oh I like your idea about the flying condition actually. You could also disable AoO when moving/charging with this condition I guess, which could be quite relevant. I don't know if it's possible to let the character levitate above its space for the time he/she flies but everything else seems actually quite reasonable.
> 
> Missing features are:
> 
> * Claws and Bite attacks
> * Proper Facing for the Breath Weapon
> * Bonus Spells Slots (Ring of Wizardry Issue)

As you might recall I've added a hook for bonus spell slots; are you able to make use of that?

Reply:
I am sure I can :)

Thanks alot for the new anim_goal, can't wait to use them for a lot of different things :)

I did  postpone my work on the DD due to some Breath Weapon issues, will resume with the DD tomorrow and will try to add the bonus spells.